docs/develop/create-addin-commands-unified-manifest.md

```diff
@@ -0,0 +1,601 @@
+
+This article applies to both scenarios, so we use the verb "ensure" for the most part, where "ensure" might mean "create", "add", "rename", or "verify" depending on what your starting point is. For example, "Ensure that there is a 'whatever' property in the 'something' object" means "Verify that there is a 'whatever' ..." when there already is a "whatever" property; but it means "Add a 'whatever' property ..." if there isn't one already.
+
+## Two decisions
+
+Decide which of two types of add-in commands you need: [Task pane or function](../design/add-in-commands.md#types-of-add-in-commands). Decide which kind of UI element you need: button or menu item. Then carry out the steps in the sections and subsections below that correspond to your decisions.
+
+## Add a task pane command
+
+The following subsections explain how to include a [task pane command](../design/add-in-commands.md#types-of-add-in-commands) in an add-in.
+
+### Configure the runtime for the task pane command
+
+1. Open the unified manifest and find the "extensions.runtimes" array.
+1. Ensure that there is a runtime object that has a "actions.type" property with the value "openPage". This is the type of runtime that opens a task pane.
+1. Ensure that the "requirements.capabilities" array contains an object that specifies a [Requirement Set](office-versions-and-requirement-sets.md) that supports add-in commands. For Outlook the minimum requirement set for add-in commands is [Mailbox 1.3](/javascript/api/requirement-sets/outlook/requirement-set-1.3/outlook-requirement-set-1.3).  
+
+    > [!NOTE]
+    > When support for the unified manifest is extended to other Office host applications, the minimum requirement set for add-in commands in those other hosts will be [AddinCommands 1.1](/javascript/api/requirement-sets/common/add-in-commands-requirement-sets).
+
+1. Ensure that the "id" of the runtime object has a descriptive name such as "TaskPaneRuntime".
+1. Ensure that the "code.page" property of the runtime object is set to the URL of the page that should open in the task pane, such as "https://localhost:3000/taskpane.html".
+1. Ensure that the "actions.view" of the runtime object has a name that describes the content of the page that you set in the preceding step, such as "homepage" or "dashboard". 
+1. Ensure that the "actions.id" of the runtime object has a descriptive name such as "ShowTaskPane" that indicates what happens when the user selects the add-in command button or menu item.
+1. Set the other properties and subproperties of the runtime object as shown in the following completed example of a runtime object. The "type" and "lifetime" properties are required and they always have the values shown in Outlook add-ins, which is the only host that currently supports the unified manifest.
```

Comment:
Something about the phrase "values shown in Outlook" makes me think the values are in Outlook, rather than in the example.
```suggestion
1. Set the other properties and subproperties of the runtime object as shown in the following completed example of a runtime object. The "type" and "lifetime" properties are required and they always have the given values in Outlook add-ins, which is the only host that currently supports the unified manifest.
```

Reply:
> Something about the phrase "values shown in Outlook" makes me think the values are in Outlook, rather than in the example.

Rewrote a different way.